npm/angular-signals/src/mount.ts

```diff
@@ -0,0 +1,536 @@
+import 'zone.js'
```

Comment:
I tried to import this from [core](https://angular.dev/api/core/provideExperimentalZonelessChangeDetection?tab=api) but couldn't find it in `@angular/core`. Looks like it might have been renamed to `ɵprovideZonelessChangeDetection`? Either way gave it a test here and seems to work ab8f852011